README.md

```diff
@@ -84,6 +84,7 @@ git clone https://github.com/google/googletest.git
 git clone https://github.com/google/glslang.git
 git clone https://github.com/KhronosGroup/SPIRV-Tools.git spirv-tools
 git clone https://github.com/KhronosGroup/SPIRV-Headers.git spirv-headers
+git clone https://github.com/KhronosGroup/SPIRV-Cross.git spirv-cross
```

Comment:
I've had complaints that we place code in directories with names different from original repos.   So let's go with directory name SPIRV-Cross rather than spirv-cross.  Yes, that breaks with the pattern from spirv-tools and spirv-headers, but we're starting fresh with SPIRV-Cross here.

Reply:
@Kangz isn't sure renaming is possible (something about case-insensitive filesystems - I don't see the problem myself) but said we could try.  Let me know if you want it and I'll do it as a separate set of changes to chrome, dawn, and shaderc.